File: hard/2_total_waiting_time/total_waiting_time_test.py
import unittest
from total_waiting_time import minimum_waiting_time

class Test(unittest.TestCase):
	def test_total_waiting_time(self):
		queries = [3, 2, 1, 2, 6]
		expected = 17
		actual = minimum_waiting_time(queries)
		self.assertEqual(expected, actual)

		queries = [1, 4, 5]
		expected = 6
		actual = minimum_waiting_time(queries)
		self.assertEqual(expected, actual)

File: hard/2_total_waiting_time/total_waiting_time.test.js
const  MinimumWaitingTime = require('./total_waiting_time');

describe('Test Total Waiting Time', () => {
	it('Test Case 1', () => {
		const queries = [3, 2, 1, 2, 6];
		const expected = 17;
		const actual = MinimumWaitingTime(queries);
		expect(actual).toEqual(expected);
	});

	it('Test Case 2', () => {
		const queries = [1, 4, 5];
		const expected = 6;
		const actual = MinimumWaitingTime(queries);
		expect(actual).toEqual(expected);
	});
});